designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview.component.html

```diff
@@ -4,7 +4,6 @@
         <div class="prj-head-container">
             <span class="prj-head-title">
                 {{$ctrl.browser.project.value.projectId}} | <strong>{{ $ctrl.browser.project.value.title }}</strong>
-                <a class="prj-head-edit" ng-click="$ctrl.manageProject($event)" ng-if="!$ctrl.readOnly"><strong>Edit Project</strong></a>
             </span>
```

Comment:
In the components for each publication preview we can remove the `manageProject()` functions that call those modals since we're not using them anymore.